frontend/src/pages/About.js

```diff
@@ -2,15 +2,33 @@ import React from 'react'
 
 import { Banner } from '../components/shared'
 import { Member, Mission, Value } from '../components/about'
+import { Grid } from '@mui/material'
 
 function About() {
     return (
-        <div>
-            <Banner />
-            <Mission />
-            <Value />
-            <Member />
-        </div>
+        <Grid container>
+
+            <Grid item xs={12}>
+                <Banner 
+                    header='About Us' 
+                    body='Making technology accessible and useful for those who create communities.'
+                    buttonText='Learn more'
```

Comment:
button should scroll to the adjacent section when click, we can maybe pass the href value for the button as a prop but this is not the only way to do the scrolling effect